### PR TITLE
doc: add a explanation for mroonga_query_log_file

### LINKED ISSUE #254: mroonga_query_log_file isn't documented

### PR DESCRIPTION
**Pull request by komainu8**
fix #254

### CODE REVIEW

**Comment by komainu8** [comment]
@kenhys Could you review this PR, please?

File: doc/source/reference/server_variables.rst [hunk 1]
```diff
@@ -392,6 +392,34 @@ Here is an example transcript to change log level to ``DEBUG`` that logs many me
   +-------------------+-------+
   1 row in set (0.00 sec)
 
+.. _serer-variable-mroonga-query-log-file:
+
```
Comment (kenhys) [inline doc/source/reference/server_variables.rst]: `_serer` -> `_server`?

File: doc/locale/ja/LC_MESSAGES/reference.po [hunk 2]
```diff
@@ -1356,6 +1356,19 @@ msgstr ""
...
 "のたくさんのメッセージがログに出力されます::"
 
+msgid "``mroonga_query_log_file``"
+msgstr ""
+
+msgid ""
+"The path of the query log file of Mroonga. The default value is a white "
+"space."
+msgstr "Mroongaのクエリーログファイル名。デフォルト値は空白です。"
+
```
Comment (kenhys) [inline doc/locale/ja/LC_MESSAGES/reference.po]: 空白 というか空では？挙動がどうかを書いたほうが素直かも。ログが作られる or 作られないみたいな。

**Comment by komainu8** [comment]
@kenhys Thank you for your comments.
I've fixed them.

**Comment by kou** [comment]
We need to update `.po`.

File: doc/source/reference/server_variables.rst [hunk 3]
```diff
@@ -392,6 +392,37 @@ Here is an example transcript to change log level to ``DEBUG`` that logs many me
...
 
+.. _server-variable-mroonga-query-log-file:
+
+``mroonga_query_log_file``
+--------------------------
+
+The path of the query log file of Mroonga. The default value is empty.
+
+If this value is empty, the query log is not stored to file.
+If this value is not empty, query log stores to the specified file.
```
Comment (kou) [inline doc/source/reference/server_variables.rst]: "is stored"

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Pull request merged**